content/influxdb/v2.2/reference/release-notes/influxdb.md

```diff
@@ -25,12 +25,19 @@ This release includes the following new [features](#features) and several [bug f
 
 #### Technical preview: replicate data remotely
 
-- Add the option to [Replicate data from InfluxDB OSS to InfluxDB Cloud](/influxdb/v2.2/write-data/replication).
+Add the option to [replicate data from InfluxDB OSS to InfluxDB Cloud](/influxdb/v2.2/write-data/replication).
 
-  This [**technical preview**](/influxdb/v2.2/reference/glossary/#technical-preview) feature lets you create a durable subscription on a per bucket basis for the purposes of replicating data on-write from an InfluxDB OSS instance to InfluxDB Cloud.
-  - Lets you store, analyze, and aggregate data locally while also forwarding newly arriving data to a centralized InfluxDB Cloud account.
-  - If InfluxDB OSS loses connectivity to InfluxDB Cloud, the data gets sent when network connectivity is reestablished.
-  - Configuration options include a maximum buffer size and data age restrictions to restrict the amount of data stored on disk.
+{{% warn %}}
+**Known issue:**
+On rare occasions, remote write failures cause data in the replication queue to get stuck. You'll be able to identify this is happening by checking the presence of data you expect in the replication target bucket. A less exact indication that this may be happening would be a buildup of the queue size. You can view this size with `influx replication list`.
```

Comment:
if this only happens on rare occasions, are we recommending they periodically check the replication target bucket programmatically? (Ideally, we want to be as explicit about how to do this proactively)....rather than say, _occurs rarely, but you'll need to figure out when to check_

For example:
On rare occasions, remote write failures cause data in the replication queue to get stuck. To prevent this issue from occurring, we recommend that you do the following <insert recommended cadence, depending on the amount of data being written?…> :
- Verify the expected data appears in the replication target bucket.
- View this queue size (of the replication target bucket?) using `influx replication list`. (If this queue exceeds ?,

Reply:
We could do that but I'm also a little worried about exposing that we don't have a good way to monitor this yet. If you think the benefit of being helpful outweighs the optics cost here, I'm good with it.

If we do it, the recommended cadence is going to be hard to pin down but we could say something like, "cadence is up to. Just keep in mind your ingest rate and how fast the queue bytes will build up given that."

Reply:
I believe the more straightforward we are (to the extent possible), the better. 
Regarding optics costs, I don't think we're fooling anyone. Ambiguous text is apt to generate questions/confusion. Our best gauge --can support team address this issue if need be.